core/store/src/db.rs

```diff
@@ -82,10 +82,12 @@ pub enum DBCol {
     ColTrieChanges = 36,
     /// Merkle tree of block hashes
     ColBlockMerkleTree = 37,
+    /// Block ordinals.
+    ColBlockOrdinal = 38,
```

Comment:
Does it have GC logic? /cc @Kouprin

Reply:
We should introduce practice to explain why we don't GC the column by writing down a comment in GC with explanation. I hope it should be easier after https://github.com/nearprotocol/nearcore/issues/2732.

Reply:
This is not expected to be GC'ed